skimage/restoration/_denoise.py

```diff
@@ -64,6 +68,30 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     >>> noisy = np.clip(noisy, 0, 1)
     >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
     """
+    if multichannel:
+        if image.ndim != 3:
```

Comment:
This will actually be a confusing message if the input image is e.g. 4D, 3D + channels. So I recommend this message for image.ndim == 2, and a different message for image.ndim > 3.